docs/reference/6-data-import/whats-new-in-data-import.md

```diff
@@ -0,0 +1,197 @@
+    *   [DI-1132](https://tracker.ed-fi.org/browse/DI-1132) - DELETE by id
+    *   [DI-1318](https://tracker.ed-fi.org/browse/DI-1318) - DELETE by natural key
+
+# What's New in Data Import v2.1
+
+The following improvements and fixes were made in Data Import v2.1:
+
+*   [DI-1008](https://tracker.ed-fi.org/browse/DI-1008) - Update DI versioning/release process to match Ed-Fi modern release practice
+*   [DI-1176](https://tracker.ed-fi.org/browse/DI-1176) - RestSharp Vulnerability & and other high priority dependabot alerts
+*   [DI-1180](https://tracker.ed-fi.org/browse/DI-1180) - Change PGBOUNCER\_LISTEN\_PORT variable in docker settings
+*   [DI-1210](https://tracker.ed-fi.org/browse/DI-1210) - Remove duplicate column validation for manual uploads
+*   [DI-1242](https://tracker.ed-fi.org/browse/DI-1242) - Move AllowUserRegistration to appsettings
+*   [DI-124](https://tracker.ed-fi.org/browse/DI-1249)[9](https://tracker.ed-fi.org/browse/DI-1249) - Error log not displaying row number during transformload run on docker
+*   [DI-126](https://tracker.ed-fi.org/browse/DI-1265)[5](https://tracker.ed-fi.org/browse/DI-1265) - Data Import v 1.3.2 does not log missing lookup failures
+*   [DI-1277](https://tracker.ed-fi.org/browse/DI-1277) - Serilog for Ingestion Log
+*   [DI-12](https://tracker.ed-fi.org/browse/DI-1282)[78](https://tracker.ed-fi.org/browse/DI-1278) - Implement ability for account lockout and reset
+*   [DI-1281](https://tracker.ed-fi.org/browse/DI-1281) - Combine Data Import web package and installer
+*   [DI-1249](https://tracker.ed-fi.org/browse/DI-1249) - Error log not displaying row number during transformload run on docker
+*   [DI-1258](https://tracker.ed-fi.org/browse/DI-1258) - Update or Replace System.Data.SqlClient in Data Import
+*   [DI-1292](https://tracker.ed-fi.org/browse/DI-1292) - 'Last Executed' row is not updated after to run the transform load
+
+# What's New in Data Import v2.0
+
+The following improvements and fixes were made in Data Import v2.0:
+
+*   [DI-1169](https://tracker.ed-fi.org/browse/DI-1169) - Multi-threading support for Data Import
+*   [DI-1230](https://edfi.atlassian.net/wiki/display/EDFITOOLS/What%27s+New) - Open sourcing of Data Import v2.0 with an Apache 2.0 license
+*   [DI-1236](https://tracker.ed-fi.org/browse/DI-1236) - Implement SSO via OpenID Connect (OIDC) (an EdWire community contribution)
+*   [DI-1238](https://tracker.ed-fi.org/browse/DI-1238) - Removing product improvement
+*   [DI-1237](https://tracker.ed-fi.org/browse/DI-1237) - Improvements to Data Import documentation to its source repository
+*   [DI-1244](https://tracker.ed-fi.org/browse/DI-1244) - Moving templates from the Template Sharing Service to an [Ed-Fi Exchange repository](https://github.com/Ed-Fi-Exchange-OSS/DataImport-Templates)
+*   [DI-1201](https://tracker.ed-fi.org/browse/DI-1201) - Implement GitHub code linting for DataImport.Common
```

Comment:
In other what's new pages, I deleted the Jira link while keeping the regular text.